spec/index.md

```diff
@@ -2114,6 +2114,24 @@ This function returns fee percentiles, measured in millisatoshi/vbyte (1000 mill
 
 The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method), inclusive, with the addition of a 0th percentile is used. Concretely, for any i from 1 to 100, the ith percentile is the fee with rank `⌈i * 100⌉`. The 0th percentile is defined as the smallest fee (excluding coinbase transactions).
 
+### IC method `metrics` {#ic-metrics}
+
+:::note
+
+The metrics management canister API is considered EXPERIMENTAL. Canister developers must be aware that the API may evolve in a non-backward-compatible way.
+
```

Comment:
```suggestion

Calling this API charges the method invocation cost to the source canister. Therefore the size of the response must be kept small. 
```

Reply:
I believe this remark applies in general and should be clear from the page on cycle costs. Hence, let's close this thread.